pymc3/math.py

```diff
@@ -14,6 +14,13 @@
 import numpy as np
 # pylint: enable=unused-import
 
+def tround(*args, **kwargs): 
```

Comment:
t for temp? because if it's for the theano it's a misnomer as all functions in here are theano functions.